contracts/contracts/token/AuctionnableTokenInterface.sol

```diff
@@ -0,0 +1,76 @@
+pragma solidity ^0.5.8;
+
+contract AuctionnableTokenInterface {
```

Comment:
Why can that not just be a erc20 token interface?

Reply:
Because I felt it was more work to find a clean 0.5.8 erc20 token interface.

Reply:
I dont understand. The only thing that does not seam to be erc20 token is the name. 
Can we not rename it to `ERC20TokenInterface` or `TokenInterface`
I just dont know what a` AuctionnableTokenInterface` should be.

Reply:
Isn't there some missing functions in the interface to comply with the erc20 token?